testing/python/metafunc.py

```diff
@@ -973,7 +973,7 @@ def pytest_generate_tests(metafunc):
         """))
         sub1.join("test_in_sub1.py").write("def test_1(): pass")
         sub2.join("test_in_sub2.py").write("def test_2(): pass")
-        result = testdir.runpytest("-v", "-s", sub1, sub2, sub1)
```

Comment:
@nicoddemus @tomviner I had to change this test because the patch changed the default behavior.